orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
My main concern here is that by hiding test data behind a wall of magic, it's going to be more difficult to debug and edit tests, and I don't trust Hypothesis to follow code updates perfectly or cover the space of data it promises to cover.  Especially when it comes to more complex objects.